sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -59,6 +59,7 @@ export class CertificateClient {
     deleteCertificateOperation(certificateName: string, options?: DeleteCertificateOperationOptions): Promise<CertificateOperation>;
     deleteContacts(options?: DeleteContactsOptions): Promise<CertificateContact[] | undefined>;
     deleteIssuer(issuerName: string, options?: DeleteIssuerOptions): Promise<CertificateIssuer>;
+    downloadCertificate(certificateName: string, version?: string): Promise<Uint8Array> // What should this return?
```

Comment:
You don't need to implement this. It's intended for .NET only, since it returns a .NET-specific type. I mean, if you have something similar that customers really want, you could add it. It just wasn't intended for other languages. We can diverge in meaningful ways. 😊